addons/mail/static/src/components/message/message.xml

```diff
@@ -178,7 +178,7 @@
                             </ul>
                         </t>
                     </div>
-                    <t t-if="message.subject and !message.isSubjectSimilarToOriginThreadName and threadView and threadView.thread and (threadView.thread.mass_mailing or [env.messaging.inbox, env.messaging.history].includes(threadView.thread))">
+                    <t t-if="message.subject and !message.isSubjectSimilarToOriginThreadName and threadView and threadView.thread and (threadView.thread.mass_mailing or [env.messaging.inbox, env.messaging.history].includes(threadView.thread) or threadView.thread.model !== 'mail.channel')">
```

Comment:
This condition is getting out of hand, don't we want to simply always display the subject now (when it's not similar to thread name)? This would make it much easier.